docs/getting-started/integrations-cdn/angular.md

```diff
@@ -97,6 +97,10 @@ All available Angular versions are [listed on npm](https://www.npmjs.com/package
 
 ## Quick start
 
+<info-box>
```

Comment:
I experimented with this callout a bit. I would probably leave the placement as it is. I tried different text versions, and "You need an account" sounds slightly less harsh than "You have to do something." The yellow color also seemed too harsh.

Reply:
WDYT about this?

```markdown
To use our CDN services, you need to [create a free account](https://portal.ckeditor.com/checkout?plan=free).
```

Reply:
Maybe the key is the simplicity. How about now?